lib/function/probability/factorial.js

```diff
@@ -55,31 +55,36 @@ module.exports = function (math) {
 
       n = n.toNumber();   // Should definitely be below Number.MAX_VALUE  
       if (n < smallBigFacs.length) {
-        return BigNumber.convert(smallBigFacs[n]).toSD(math.precision);
+        return BigNumber.convert(smallBigFacs[n]).toSD(config.precision);
       }
 
+      // Be wary of round-off errors
+      var precision = config.precision + 4;
```

Comment:
Hm that's interesting. Qua calculation time I suppose evaluating `Math.ceil(Math.log(n) / 3)` is neglectable compared to the actual iteration so that shouldn't have a negative impact. Why exactly `log(n)/3` ?